src/MDLUpgrade.js

```diff
@@ -0,0 +1,15 @@
+import { Children, Component, findDOMNode } from 'react';
+
+export default class MDLUpgrade extends Component {
```

Comment:
Sorry, I thought that if it's a component and not function which wraps component then I should put it alongside other components.